client/src/partials/stock/loss_record/loss_record.js

```diff
@@ -0,0 +1,208 @@
+angular.module('bhima.controllers')
+.controller('stock.loss_record', [
+  '$scope',
+  '$timeout',
+  '$routeParams',
+  'util',
+  'validate',
+  'exchange',
+  function ($scope, $timeout, $routeParams, util, validate, exchange) {
+    // TODO add search (filter)
+    // TODO add sortable (clickable) columns
```

Comment:
TODOs are excellent for keeping track of progress.  It's good to see you using them.